virt_who/register.py

```diff
@@ -581,6 +581,36 @@ def stage_job_delete(self, ssh, register_config, job_ids):
                 ret, output = self.runcmd(cmd, ssh)
                 logger.info("Finished to cancel job {0}".format(job_id))
 
+    def stage_sca_enable(self, ssh, register_config):
+        api = register_config['api']
+        username = register_config['username']
+        password = register_config['password']
+        owner = register_config['owner']
+        curl_header = '-H "accept: application/json" -H "Content-Type: application/json"'
+        json_data = json.dumps('{"contentAccessMode":"org_environment"}')
+        cmd = "curl -X PUT -s -k -u {0}:{1} -d {2} {3} {4}/owners/{5}".format(
+            username, password, json_data, curl_header, api, owner)
+        ret, output = self.runcmd(cmd, ssh)
+        if ret == 0 and '"contentAccessMode":"org_environment"' in output:
+            logger.info("Succeeded to enable SCA mode for stage candlepin")
+        else:
+            raise FailException("Failed to enable SCA mode for stage candlepin")
+
+    def stage_sca_disable(self, ssh, register_config):
+        api = register_config['api']
+        username = register_config['username']
+        password = register_config['password']
+        owner = register_config['owner']
+        curl_header = '-H "accept: application/json" -H "Content-Type: application/json"'
+        json_data = json.dumps('{"contentAccessMode":"entitlement"}')
+        cmd = "curl -X PUT -s -k -u {0}:{1} -d {2} {3} {4}/owners/{5}".format(
+            username, password, json_data, curl_header, api, owner)
+        ret, output = self.runcmd(cmd, ssh)
+        if ret == 0 and '"contentAccessMode":"entitlement"' in output:
+            logger.info("Succeeded to disable SCA mode for stage candlepin")
+        else:
+            raise FailException("Failed to disable SCA mode for stage candlepin")
+
```

Comment:
Actually, you could reduce the 2 functions to:
```
    def stage_sca_set(self, ssh, register_config, enable=True):
        api = register_config['api']
        username = register_config['username']
        password = register_config['password']
        owner = register_config['owner']
        curl_header = '-H "accept: application/json" -H "Content-Type: application/json"'
        if enable:
            body = {"contentAccessMode": "org_environment"}
        else:
            body = {"contentAccessMode": "entitlement"}
        json_data = json.dumps(body)
        cmd = "curl -X PUT -s -k -u {0}:{1} -d {2} {3} {4}/owners/{5}".format(
            username, password, json_data, curl_header, api, owner)
        ret, output = self.runcmd(cmd, ssh)
        if ret == 0 and f'''"contentAccessMode":"{body['contentAccessMode']}"''' in output:
            logger.info(f"Succeeded to set SCA mode for stage candlepin to {enable}")
        else:
            raise FailException(f"Failed to set SCA mode for stage candlepin to {enable}")
```

Reply:
Just a small idea, you could ignore it if you think yours are more readable :)

Reply:
Yeah, it's a good idea, at the beginning I didn't have a good idea to check the output to make the code more clear.
After checking I deleted this checking because the ```ret == 0``` can do the work.

Reply:
Great :+1: